src/test/resources/features/08_additional_information.feature

```diff
@@ -143,4 +143,235 @@ Feature: Additional Information Journeys (Interest Securities) - Income Tax Subm
+      |Affinity Group     |Individual                                           |
+      |Nino               |AA123456A                                            |
+      |Enrolment Key 1    |HMRC-MTD-IT                                          |
+      |Identifier Name 1  |MTDITID                                              |
+      |Identifier Value 1 |1234567890                                           |
+    Then the user should be redirected to the "Update and submit an Income Tax Return" page
+    When the user clicks the continue button
+    Then the user should be redirected to the "Your Income Tax Return" page
+    When the user clicks on the addSections link
+    Then the user should be redirected to the "Add sections to your Income Tax Return" page
+    When the user clicks the addGains tailoring option
+    And the user clicks the continue button
+    Then the user should be redirected to the "Your Income Tax Return" page
+    When the user clicks on the Gains from policies and contracts link
+    Then the user should be redirected to the "Gains from life insurance policies and contracts" page
+    When the user selects the yes radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "What type of policy gave you a gain?" page
+    When the user selects the Life Insurance radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Policy number" page
+    When the user selects the Policy number field and enters a value of LA/2881/07
+    And the user clicks the continue button
+    Then the user should be redirected to the "Chargeable event gain" page
+    When the user selects the amount field and enters a value of 123.45
+    And the user clicks the continue button
+    Then the user should be redirected to the "Policy event" page
+    When the user selects the Sale or assignment of a policy radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Have you had a gain from this policy in an earlier tax year?" page
+    When the user selects the yes radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Years since your last gain" page
+    When the user selects the Year field and enters a value of 99
+    And the user clicks the continue button
+    Then the user should be redirected to the "Policy held" page
+    When the user selects the Year field and enters a value of 99
+    And the user clicks the continue button
+    Then the user should be redirected to the "Was your gain treated as tax paid?" page
+    When the user selects the no radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Are you entitled to deficiency relief?" page
+    When the user selects the no radio button
+    And the user clicks the continue button
```

Comment:
Agent user flow and prior submission tests required

Reply:
just added